### ISSUE TITLE
Update report Object `name` and `description` value content

### ISSUE THREAD

**Issue by himynamesdave** [signalscorps/stixify#10]
Currently the `name` value prints full local path to report.

e.g.

```
        {
            "type": "report",
            "spec_version": "2.1",
            "id": "report--56f2fbd1-3dba-4990-95f5-137255b728c0",
            "created": "2022-08-16T06:21:22.041605Z",
            "modified": "2022-08-16T06:21:22.041605Z",
            "name": "/Documents/GitHub/obstracts-cli/examples/input.txt",
            "report_types": [
                "threat_report"
            ],
            "published": "2022-08-16T07:21:22.041533Z",
```

It should print only the filename prefixed with `File converted: <FILENAME>"

In the `description` Object, should print the extracted text of the report (with json escaped)

**Comment by himynamesdave** [comment]
@shanthanu9 in fact, just leave the description (do not print it).

e.g.

```json
        {
            "type": "report",
            "spec_version": "2.1",
            "id": "report--56f2fbd1-3dba-4990-95f5-137255b728c0",
            "created": "2022-08-16T06:21:22.041605Z",
            "modified": "2022-08-16T06:21:22.041605Z",
            "name": "input.txt",
            "report_types": [
                "threat_report"
            ],
            "published": "2022-08-16T07:21:22.041533Z",
```

@shanthanu9

**Comment by himynamesdave** [comment]
@shanthanu9 in fact, just leave the description (do not print it).

e.g.

```json
        {
            "type": "report",
            "spec_version": "2.1",
            "id": "report--56f2fbd1-3dba-4990-95f5-137255b728c0",
            "created": "2022-08-16T06:21:22.041605Z",
            "modified": "2022-08-16T06:21:22.041605Z",
            "name": "input.txt",
            "report_types": [
                "threat_report"
            ],
            "published": "2022-08-16T07:21:22.041533Z",
```

@shanthanu9

**Comment by himynamesdave** [comment]
@shanthanu9 name still includes full filepath

```json
        {
            "type": "report",
            "spec_version": "2.1",
            "id": "report--252a18dd-0b46-4985-a102-a2c09ee37e8a",
            "created": "2022-08-24T18:15:09.529175Z",
            "modified": "2022-08-24T18:15:09.529175Z",
            "name": "/Documents/GitHub/stixify/tests/file_inputs/txt/input.txt",
            "report_types": [
                "threat_report"
            ],
            "published": "2022-08-24T19:15:09.529064Z",
            "object_refs": [
```
Should just read filename, e.g `"name": "input.txt",`
